dev-util/pycharm-professional/metadata.xml

```diff
@@ -4,5 +4,6 @@
 <!-- maintainer-needed -->
 <use>
 	<flag name="bundled-jdk">Use bundled jdk</flag>
+	<flag name="bundled-xvfb">Use bundled Xvfb</flag>
```

Comment:
Is there more to this, I'm looking for the why anyone would ever choose this option?